File: VB/TransparentEditorExample/Form1.vb
Imports Microsoft.VisualBasic
Imports System
Imports DevExpress.XtraEditors

Namespace TransparentEditorExample
	Partial Public Class Form1
		Inherits XtraForm
		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub Form1_Load(ByVal sender As Object, ByVal e As EventArgs) Handles MyBase.Load
			' TODO: This line of code loads data into the 'nwindDataSet.Customers' table. You can move, or remove it, as needed.
			Me.customersTableAdapter.Fill(Me.nwindDataSet.Customers)

		End Sub
	End Class
End Namespace

File: CS/TransparentEditorExample/Form1.cs
using System;
using DevExpress.XtraEditors;

namespace TransparentEditorExample {
    public partial class Form1 :XtraForm {
        public Form1() {
            InitializeComponent();
        }

        private void Form1_Load(object sender, EventArgs e) {
            // TODO: This line of code loads data into the 'nwindDataSet.Customers' table. You can move, or remove it, as needed.
            this.customersTableAdapter.Fill(this.nwindDataSet.Customers);

        }
    }
}